content/doc/book/blueocean/dashboard.adoc

```diff
@@ -19,105 +19,78 @@ endif::[]
 
 = Dashboard
 
-Blue Ocean's "Dashboard" is the default view shown when you open Blue Ocean and
-shows an overview of all Pipeline projects configured on a Jenkins instance.
+Blue Ocean's *Dashboard* is the default view shown when opening Blue Ocean.
```

Comment:
I am a little uncertain about the *Dashboard* phrasing, since I do not see the word Dashboard in the UI. I think that they put the quotes around it because that is how they refer to it when it isn't something that can be selected in the UI. 

Here is a note from Microsoft manual of style about the term "dashboard": https://docs.microsoft.com/en-us/style-guide/word-choice/use-technical-terms-carefully

And why I chose "user interface" for my suggestion:
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/i/interface

```suggestion
Blue Ocean's user interface is the default view when opening Blue Ocean.
```

Reply:
discussed on Slack, leaving Dashboard terminology as it is describing the _intended_ purpose/definition of the page, despite Dashboard not being a part of the UI.